pages/ox_lib/Modules/Logger/Server.mdx

```diff
@@ -85,3 +85,21 @@ It is designed to be very cost effective and easy to operate.
   set loki:password "<insert the api key or password>"
   set loki:endpoint "<insert loki url without http:// or https://>"
   ```
+
+
+## Fivemanage
+Fivemanage is an easy-to-use, all-in-one SaaS for game servers that provides logging.
```

Comment:
for some reason I am not confident that the target audience here is going to understand the term `SaaS`